### ISSUE TITLE
No TLD necessary for valid email

### ISSUE THREAD

**Issue by Datamine** [syrusakbary/validate_email#55]
e.g.

```
>from validate_email import validate_email
>validate_email("foo@gmail")
True
```

The regex should check for a valid TLD.

**Comment by ghost** [comment]
Except a TLD is not required for a valid email address. Sometimes email addresses are used on LANs or locally, for example `foo@localhost` or `bar@localdomain`.

Maybe adding a parameter like `require_tld` would work?

```
>validate_email("foo@gmail", require_tld=True)
False
```

**Comment by syrusakbary** [comment]
Yeah, I think this is a good idea (the `require_tld` argument)

**Comment by Necior** [comment]
What about server part being an IP address (either IPv4 or IPv6)? For example `user@198.51.100.13` is a valid email.

**Comment by HaraldNordgren** [comment]
Definitely add a `require_tld` argument. 🎉

**Comment by ferret-guy** [comment]
Created a pull request to address this

**Comment by racitup** [comment]
Yep, I need this to check for potentially publicly routable email addresses please
